From b0daa96e765d1c73cf48673bb57ce7940231b79a Mon Sep 17 00:00:00 2001
From: Erik Faye-Lund <kusmabite@gmail.com>
Date: Tue, 28 Sep 2010 18:05:26 +0200
Subject: [PATCH/RFC 1/2] gettext: use const char* instead of char*
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Change the charset variable in git_setup_gettext from char* to const
char*.

Signed-off-by: Erik Faye-Lund <kusmabite@gmail.com>
Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 gettext.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/gettext.c b/gettext.c
index 8644098..d20bb39 100644
--- a/gettext.c
+++ b/gettext.c
@@ -7,7 +7,7 @@
 extern void git_setup_gettext(void) {
 	char *podir;
 	char *envdir = getenv("GIT_TEXTDOMAINDIR");
-	char *charset;
+	const char *charset;
 
 	if (envdir) {
 		(void)bindtextdomain("git", envdir);
-- 
1.7.3.159.g610493

